file-formats/suco/type/zif_aff_suco_v1.intf.abap

```diff
@@ -0,0 +1,44 @@
+"! AFF Type for Object SUCO (Authorization Default Variant)
+INTERFACE zif_aff_suco_v1
+  PUBLIC.
+
+  TYPES:
+    "! <p class="shorttext">Leading Application</p>
+    "! Leading application
+    BEGIN OF ty_leading_application,
+      "! <p class="shorttext">Program ID</p>
+      "! Program ID
+      "! $required
+      program_id   TYPE c LENGTH 4,
```

Comment:
Minor: Do you really need the program ID for "R3TR" or "LIMU"? Couldn't this been calculated from the object type?

Reply:
The program ID could be determined, but we want to display this information in the editor. Therefore we would need to keep the program ID in the AFF.

Reply:
I guess you know my point of view ;): I would remove it. But I think it is your decision. 

If you want to keep it, maybe, you want to consider to remove the `$required` and even set a default value to "R3TR".
In this case, the JSON (per default) would not contain the program id, if it is "R3TR" which is most probably the case for most of the cases (except the very prominent example of LIMU FUNC. What do you think?

Reply:
Hi @schneidermic0 and @GuilhermeSaraiva96 , I removed the $required and set the default to "R3TR".